proposals/000-ensure-compilation-scalability.md

```diff
@@ -8,17 +8,19 @@ Proposed by Stu Hood, Eugene Burmako - Twitter - November 2016
 
 The most commonly discussed angle of attack for improving the Scala build experience is speeding up the compiler. Instead, this proposal addresses a different angle: minimizing compilation classpaths to enable distributed builds.
 
-## Minimization of compilation classpaths
+## Background
 
-Our build workflow at Twitter is centered around a monorepo. Instead of keeping internal projects in separate repositories, we merge them into one repository (hence the name “monorepo”), significantly reducing the overhead associated with evolving independent repositories.
+### Minimization of compilation classpaths
 
-In comparison with a more common workflow that involves a number of coarse-grained projects and intra-project incremental compilation via sbt, our build workflow relies on many more (tens of thousands) fine-grained modules and inter-project incrementality. This workflow is enabled by Pants, our open-source build tool that understands a multitude of languages including Java, Scala, Python and others.
+Our build workflow at Twitter is centered around a monorepo. Instead of keeping internal projects in separate repositories, we merge them into one repository (hence the name "monorepo"), significantly reducing the overhead associated with evolving independent repositories.
```

Comment:
I don't think this is relevant to the problem, so it might be a bit distracting. The problem occurs with very large projects, regardless of monorepoism.

Would drop monorepo references and/or replace them with mentions of "very large projects".